src/App.svelte

```diff
@@ -1,10 +1,13 @@
 <script>
   import Router from 'svelte-spa-router';
-  import { routes } from './routes/index.js';
+  import { routes } from './routes/index';
   import Header from './components/Header.svelte';
+  import { isLoading } from 'svelte-i18n';
 </script>
 
 <main>
-  <Header />
-  <Router {routes} />
+  {#if !$isLoading}
+    <Header />
+    <Router {routes} />
+  {/if}
```

Comment:
can we add some simple else block with centred gif loading animation like this ? https://css-loaders.com/spinner/ (should be separate component)